desktop/components/react/utils/template_renderer.js

```diff
@@ -20,22 +28,26 @@ export function makeTemplate (templates, options = {}) {
     ? templates
     : [templates]
 
-  const basePath = options.basePath || ''
+  const {
+    basePath,
+    compilerOptions,
+    locals
+  } = merge(defaultOptions, opts)
 
   const templateFns = templates.map(file => {
     return jade.compileFile(path.join(basePath, file), {
       cache: true,
-      ...options
+      ...compilerOptions
     })
   })
 
-  return {
-    render: (locals = {}) => {
-      const renderTemplate = locals => template => template(locals)
+  const renderTemplate = locals => template => template(locals)
+  const renderedTemplates = templateFns.map(renderTemplate(locals))
+
+  // If user only passed in a single template, return a single template
```

Comment:
This could be surprising if that single template was in an array, especially in a dynamic usage where the size was unknown.